app/js/newsfeed.js

```diff
@@ -148,6 +148,7 @@ module.exports = {};
       });
   }
 
+
```

Comment:
Extra line also seems unnecessary here (unless 2 lines before the exports was intentional?)